server/src/mindustry/server/ServerControl.java

```diff
@@ -172,6 +172,18 @@ public ServerControl(String[] args){
             }
         });
 
+        Events.on(WaveEvent.class, e -> {
+            if(state.is(State.playing) && Config.valueOf("enableAutosave").bool() ){
+                Fi file = saveDirectory.child("autosave-" + Strings.capitalize(world.getMap().name()) + "-" + state.wave + "." + saveExtension);
+
+                Core.app.post(() -> {
+                    SaveIO.save(file);
+                    Call.sendMessage("[scarlet][Server]:[] - Autosave Done");
+                    info("&lyServer: &lb{0}", "Autosave Done");
```

Comment:
Why prefix this with `Server:`? The console log isn't a chat, and this formatting seems unnecessary since the argument is always the same. Just print `info("&lbAutosave completed.")`.